src/peerd/runtime.rs

```diff
@@ -314,12 +311,18 @@ impl esb::Handler<ServiceBus> for Runtime {
         endpoints: &mut Endpoints,
         bus: ServiceBus,
         source: ServiceId,
-        request: Request,
+        request: BusMsg,
     ) -> Result<(), Self::Error> {
-        match bus {
-            ServiceBus::Msg => self.handle_rpc_msg(endpoints, source, request),
-            ServiceBus::Ctl => self.handle_rpc_ctl(endpoints, source, request),
-            ServiceBus::Bridge => self.handle_bridge(endpoints, source, request),
+        match (bus, request) {
+            // Peer-to-peer message bus
+            (ServiceBus::Msg, BusMsg::Msg(req)) => self.handle_msg(endpoints, source, req),
+            // Control bus for issuing internal commands
+            (ServiceBus::Ctl, BusMsg::Ctl(req)) => self.handle_ctl(endpoints, source, req),
+            // User issued command RPC bus, only accept BusMsg::Rpc
+            (ServiceBus::Rpc, BusMsg::Rpc(req)) => self.handle_rpc(endpoints, source, req),
```

Comment:
these comments don't match how the buses are used anymore

Reply:
Done in [f28c0b2](https://github.com/farcaster-project/farcaster-node/pull/707/commits/f28c0b26a8a366ff8ad73e6285c37ae985c9a76c)